client/src/components/Account/Account.jsx

```diff
@@ -9,19 +10,48 @@ import { AppContext } from "../../context/Context";
 
 function Account() {
     const { user, bankAccounts, setBankAccounts, houseMembers, setHouseMembers, error, setError } = AppContext();
+    const navigate = useNavigate();
 
-    const accountList = bankAccounts ? bankAccounts.map((account) => {
+    const accountList = bankAccounts ? bankAccounts?.map((account) => {
         return <BankItem key={account.id} account={account} setBankAccounts={setBankAccounts} setError={setError} />
     }) : null;
 
-    const householdList = houseMembers ? houseMembers.map((member) => {
+    const householdList = houseMembers ? houseMembers?.map((member) => {
         return <HouseMember key={member.id} houseMember={member} />
     }) : null;
 
     return (
         <>
-            <h1>Account Management</h1>
-            <Typography>Linked Accounts</Typography>
+            <h1
+            style={{
+                textAlign: 'center', 
+                color: 'green', 
+                fontSize: '2.5rem', 
+                margin: '1rem 0'
+            }}
+            >Account Management</h1>
+            <Typography
+            style={
+                {textAlign: 'center', 
+                color: 'green', 
+                fontSize: '1.5rem', 
+                margin: '1rem 0'}
+            }
+            >Linked Bank Accounts</Typography>
```

Comment:
With the new PlaidButton component, we don't need to route to /plaid.  The button itself can be imported and used in any other component.  We should use the PlaidButton component here, but the PlaidButton should be changed to have this styling.